### DISCUSSION TITLE
代码片段功能可以增加分组吗

### DISCUSSION THREAD

**Discussion by rianjskis** [lollipopkit/flutter_server_box#257] [Ideas]
需要保存的片段有点多，如果没有分组的话，很不方便

**Comment by lollipopkit** [comment]
当前版本代码片段有标签功能，这就是用来分组的
![image](https://github.com/lollipopkit/flutter_server_box/assets/10864310/b94c4f8f-7725-4c52-acf1-d79f997c0723)
